src/simplay-jupyter/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Compare with https://github.com/team-simplay/SimPlay/blob/main/src/simplay-web/tsconfig.json it should be clear why we differ, maybe this could also be part of the Bericht, if so please create a task for it.

Reply:
task created: https://gitlab.ost.ch/david.kuehnhanss/ba_simpy_animationlib/-/issues/77